package.json

```diff
@@ -66,7 +66,7 @@
     "npm-run-all": "^4.1.5",
     "object.assign": "~4.1.0",
     "package-json-reducer": "^1.0.18",
-    "prettier": "^2.3.0",
+    "prettier": "~2.3.0",
```

Comment:
While working on this, I ran into an issue where a later version of prettier caused a whole mess of TypeScript issues. See [this issue](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/60314) for more info. We need to keep this version here until we can upgrade our TypeScript version (which has always caused other issues).